### DISCUSSION TITLE
Duplicated upload

### DISCUSSION THREAD

**Discussion by hasbai** [Chevereto/Chevereto-Free#159] [General]
As far as I know that if an IP uploads the same image within 24 hours, chevereto will return 'Duplicated upload' error.
```json
{
    "status_code": 400,
    "error": {
        "message": "Duplicated upload",
        "code": 102,
        "context": "Exception"
    },
    "status_txt": "Bad Request"
}
```
It's inconvenient when use it as an image backend in my project. It's preferred that chevereto will return the existed url of the image instead if it is duplicated. Is there any way to implement this feature?

**Comment by rodber** [comment]
The function for dupes is [CHV\Image::isDuplicatedUpload](https://github.com/rodber/chevereto-free/blob/11718635a4b223dc3ebec94dd60d9dc72630fe77/app/lib/classes/class.image.php#L597). In this method you can see how the systems detects an existing image.

From there you need to intercept here: https://github.com/rodber/chevereto-free/blob/11718635a4b223dc3ebec94dd60d9dc72630fe77/app/lib/classes/class.image.php#L646

And then dummy the return based on your use case.
